Documentation/project-docs/writing-tests.md

```diff
@@ -0,0 +1,93 @@
+This document provides an overview of paradigm we are using in our tests.
+
+# RemoteExecutor
```

Comment:
> What do you mean by this? How could it be a .NET Framework app if it runs on Linux and macOS, as it does?

This is based on an internal discussion with @weshaggard and the fact that the binary fails strong-name signing. Attaching a debugger when this is ran in UWP shows that it's referencing mscorlib which I thought we don't have in CoreFX.